scripts/healthcheck.sh

```diff
@@ -23,15 +23,15 @@ echo "Network is up"
 
 #Service check
 #Expected output is 2 for both checks, 1 for process and 1 for grep
-OPENVPN=$(ps -ef | grep 'openvpn --script-security' |wc| awk '{print $1}')
-TRANSMISSION=$(ps -ef | grep 'transmission-daemon' |wc| awk '{print $1}')
+OPENVPN=$(ps -ef | grep 'openvpn --script-security' | grep -v 'grep openvpn --script-security' | wc | awk '{print $1}')
+TRANSMISSION=$(ps -ef | grep 'transmission-daemon' | grep -v 'grep transmission-daemon' | wc | awk '{print $1}')
```

Comment:
can also achieve the same with, as an example:

```OPENVPN=$(ps -ef | grep '[o]penvpn --script-security' | wc -l )```

Much less pipes on pipes on pipes :)